blockchain/blockchain-wallet-v4-frontend
# feat(Browser deauthorization support)

## Description
Adds ability for users to deauthorize their browsers which will require them to verify email and 2FA upon next login.  A 10 second timer is started when the `/logout` route is hit. If the user doesn't make a decision in that time, they will be redirected to `/login`.

![screen shot 2018-06-11 at 6 51 57 pm](https://user-images.githubusercontent.com/6364918/41261285-f27d83e8-6da8-11e8-965a-883d44d76021.png)

## Change Type
Feature

## Testing Steps
Login using a 2FA enabled wallet and upon logout, select deauthorize browser and verify 2FA will be again required.

## PR Creator Checklist
